src/inc/til/rle.h

```diff
@@ -503,6 +505,9 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             }
 
             _compact();
+
+            _hint_run = 0;
+            _hint_index = 0;

```

Comment:
Above is a function called `runs()` which is an escape hatch for a performance improvement in `ROW::Reset`. We should add a `rle` method like this:

```cpp
void fill(const value_type& new_value)
{
    _runs.clear();
    _runs.emplace_back(value, _total_length);
}
```

Afterwards we should modify `runs()` to return a const reference.